src/openforms/authentication/models.py

```diff
@@ -61,41 +68,154 @@ def hash_identifying_attributes(self, delay=False):
         verbose_name=_("Submission"),
         on_delete=models.CASCADE,
         help_text=_("Submission related to this authentication information"),
         related_name="auth_info",
     )
-    machtigen = models.JSONField(
-        verbose_name=_("machtigen"),
+
+    # authentication details of the representee (without mandate/machtigen, this is the
+    # logged in actor themselves) are stored in the fields `plugin`, `attribute` and `value`.
+    # The match the (inferred) `representee.identifierType` and `representee.identifier`.
+
+    # together with the value of `plugin`, this tracks the way authentication was
+    # performed.
+    loa = models.TextField(
+        verbose_name=_("Level of assurance"),
         help_text=_(
-            "Data related to any 'machtiging' (authorising someone else to perform actions on your behalf)."
+            "How certain is the identity provider that this identity belongs to this user."
+        ),
+        default="",  # not all plugins support this concept
+        blank=True,
+    )
+
+    # track information about the mandate/machtigen. Note that this also captures an
+    # employee authenticating for their company, they are then the acting subject and
+    # the company is the legal subject.
+    acting_subject_identifier_type = models.CharField(
+        verbose_name=_("acting subject identifier type"),
+        help_text=_(
+            "The identifier type determines how to interpret the identifier value."
+        ),
+        max_length=50,
+        blank=True,
+        choices=ActingSubjectIdentifierType.choices,
+    )
+    acting_subject_identifier_value = models.CharField(
+        verbose_name=_("acting subject identifier"),
+        help_text=_("(Contextually) unique identifier for the acting subject."),
+        max_length=250,
+        blank=True,
+    )
+
+    legal_subject_identifier_type = models.CharField(
+        verbose_name=_("legal subject identifier type"),
+        help_text=_(
+            "The identifier type determines how to interpret the identifier value."
+        ),
+        max_length=50,
+        blank=True,
+        choices=LegalSubjectIdentifierType.choices,
+    )
+    legal_subject_identifier_value = models.CharField(
+        verbose_name=_("legal subject identifier"),
+        help_text=_("(Contextually) unique identifier for the legal subject."),
+        max_length=250,
+        blank=True,
+    )
+
+    mandate_context = models.JSONField(
+        verbose_name=_("mandate context"),
+        help_text=_(
+            "If a mandate is in play, then the mandate context must be provided. The "
+            "details are tracked here, in line with the authentication context data "
+            "JSON schema definition."
         ),
         blank=True,
         null=True,
     )
-    loa = models.TextField(
-        verbose_name=_("Level of assurance"),
+
+    # deprecated!
+    machtigen = models.JSONField(
+        verbose_name=_("machtigen"),
```

Comment:
I guess there's no way to extract things from this deprecated fields to set them to the mandate/authorizee fields?

Reply:
even worse - the data stored in there is just plain wrong now :)